plugins/janus_audiobridge.c

```diff
@@ -3720,22 +3742,22 @@ void janus_audiobridge_incoming_rtp(janus_plugin_session *handle, janus_plugin_r
 		participant->inbuf = g_list_insert_sorted(participant->inbuf, pkt, &janus_audiobridge_rtp_sort);
 		if(participant->prebuffering) {
 			/* Still pre-buffering: do we have enough packets now? */
-			if(g_list_length(participant->inbuf) > DEFAULT_PREBUFFERING) {
+			if(g_list_length(participant->inbuf) > participant->prebuffer_count) {
 				participant->prebuffering = FALSE;
 				JANUS_LOG(LOG_VERB, "Prebuffering done! Finally adding the user to the mix\n");
 			} else {
 				JANUS_LOG(LOG_VERB, "Still prebuffering (got %d packets), not adding the user to the mix yet\n", g_list_length(participant->inbuf));
 			}
 		} else {
 			/* Make sure we're not queueing too many packets: if so, get rid of the older ones */
-			if(g_list_length(participant->inbuf) >= DEFAULT_PREBUFFERING*2) {
+			if(g_list_length(participant->inbuf) >= participant->prebuffer_count*2) {
```

Comment:
Here prebuffer count limits the inbuf capacit

Reply:
You're right, that value will also limit the buffer capacity (as double the value).

It could be better to configure a total buffer length and then compute a prebuffering threshold based on the total buffer length. I find the current configuration method a bit misleading (configure the pre-buffer and compute the length as 2 * pre-buffer).

Also having a buffer length upper bound (e.g. 32, equivalent to ~600 ms of audio delay) might be a good idea (this to avoid `g_list_length` and `g_list_insert_sorted` calls on longer lists, potentially hitting the performance).

Reply:
@chpl got what you meant now :+1: 
I thought you were referring to the size of the packets we buffer in the core (which we call NACK queue), which is unrelated to this value here.